build/Repositories.props

```diff
@@ -7,6 +7,7 @@
 
   <ItemGroup>
     <Repository Include="Antiforgery" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
+    <Repository Include="AspNetCoreModule" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
```

Comment:
I don't think we can do this as yet. This requires the C++ sdk which we don't have installed on all our build agents.